replace_using_package_version.service

```diff
@@ -14,6 +14,13 @@ It uses python re module syntax</description>
     <description>This is the package whos version will be used as a replacement
 for the regex. It must be included to the build as dependency.</description>
   </parameter>
+  <parameter name="match">
+    <description>This is a regular expression applied to the version of
+    of the package, if present first match is used as the replacement string.
+    If no match the full version string is used as the replacement string.
```

Comment:
If no match the program will raise an exception, iiuc the code.

Wouldn't it be better to do as stated here? To return the full version string?

Reply:
Yes I forgot to change that while doing some tests, you are right :+1: